bot/kodiak/test_evaluation.py

```diff
@@ -3466,6 +3467,30 @@ def test_get_merge_body_empty(pull_request: PullRequest) -> None:
     assert actual == expected
 
 
+def test_get_merge_body_includes_pull_request_url(pull_request: PullRequest) -> None:
```

Comment:
Can you add a test for this with the coauthor setting. The coauthor should be placed after this one.

Reply:
donezo